packages/mds-metrics-sheet/metrics-log.ts

```diff
@@ -170,8 +172,8 @@ async function getProviderMetrics(iter: number): Promise<MetricsSheetRow[]> {
           providerdropoff: event_counts.provider_drop_off || 0,
           tripstart: starts,
           tripend: ends,
-          tripenter: last[provider.provider_id].event_counts_last_24h.trip_enter || 0,
-          tripleave: last[provider.provider_id].event_counts_last_24h.trip_leave || 0,
+          tripenter: get(event_counts_last_24h, 'trip_enter', 0),
```

Comment:
Not super clear why `lodash.get` is required here.

Reply:
Agreed, I think this may be unnecessary now that we do the check for `event_counts_last_24h`. The previous logic of `event_counts_last_24h.trip_enter || 0` should still work. The initial bug was that we were attempting to get a property from an undefined element, but we're doing that no longer!